demisto_sdk/commands/validate/validators/GR_validators/GR104_is_pack_display_name_already_exists.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import annotations
+
+from abc import ABC
+from typing import Iterable, List
+
+from demisto_sdk.commands.content_graph.objects.pack import Pack
+from demisto_sdk.commands.content_graph.parsers.related_files import RelatedFileType
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Pack
+
+
+class IsPackDisplayNameAlreadyExistsValidator(BaseValidator, ABC):
+    error_code = "GR104"
+    description = "Validate that there are no duplicate display names in the repo"
+    rationale = " Validate the existance of duplicate display names"
```

Comment:
```suggestion
    description = "Validate that there are no duplicate display names in the repo."
    rationale = " We want to avoid the existance of duplicate display names in order to make sure we don't confuse between content items."
```